### ISSUE TITLE
Tabby 1.0.203: OS Default tab not closing and the tabby app wont quit

### ISSUE THREAD

**Issue by MuhtasimTanmoy** [Eugeny/tabby#9230]
**Describe the problem**:
- Once started, the Tabby app will not quit. The open tabs will not close as well.

**To Reproduce**:
- Open Tabby
- Open a few tabs
- Closing those should not work
- `Control + Q` to close the app won't work either

**Environment**
- Tabby Version: 1.0.203 (1.0.203)
- OS: macOS Sonoma 14.1.1

**Query**
- Where to look for logs so I can share?

**Comment by soobeencn** [comment]
it's same with me, help

**Comment by xDong2781** [comment]
I also encountered the same issue

**Comment by luzhaung** [comment]
same issue

**Comment by Clem-Fern** [comment]
@Eugeny I got a bit of time rn, I will try to take a look

**Comment by MuhtasimTanmoy** [comment]
> Could you please send log at the time of the issue ?

```
error_handler.ts:45 ERROR Error: Uncaught (in promise): TypeError: Cannot read properties of undefined (reading 'getProcessList')
TypeError: Cannot read properties of undefined (reading 'getProcessList')
    at ElectronPTYProxy.<anonymous> (/Applications/Tabby.…t/index.js:35620:64)
    at Generator.next (<anonymous>)
    at /Applications/Tabby.…t/index.js:25927:67
    at new ZoneAwarePromise (zone.js:1411:21)
    at tslib_es6_awaiter (/Applications/Tabby.…t/index.js:25909:10)
    at ElectronPTYProxy.getChildProcessesInternal (/Applications/Tabby.…t/index.js:35618:16)
    at ElectronPTYProxy.<anonymous> (/Applications/Tabby.…t/index.js:35614:25)
    at Generator.next (<anonymous>)
    at fulfilled (/Applications/Tabby.…t/index.js:25912:24)
    at _ZoneDelegate.invoke (zone.js:368:26)
    at Object.onInvoke (ng_zone.ts:423:29)
    at _ZoneDelegate.invoke (zone.js:367:52)
    at Zone.run (zone.js:127:43)
    at zone.js:1257:36
    at _ZoneDelegate.invokeTask (zone.js:402:31)
    at Object.onInvokeTask (ng_zone.ts:408:29)
    at _ZoneDelegate.invokeTask (zone.js:401:60)
    at Zone.runTask (zone.js:171:47)
    at drainMicroTaskQueue (zone.js:581:35)
    at resolvePromise (zone.js:1193:31)
    at zone.js:1100:17
    at zone.js:1116:33
    at rejected (/Applications/Tabby.…st/index.js:14211:9)
    at _ZoneDelegate.invoke (zone.js:368:26)
    at Object.onInvoke (ng_zone.ts:423:29)
    at _ZoneDelegate.invoke (zone.js:367:52)
    at Zone.run (zone.js:127:43)
    at zone.js:1257:36
    at _ZoneDelegate.invokeTask (zone.js:402:31)
    at Object.onInvokeTask (ng_zone.ts:408:29)
    at _ZoneDelegate.invokeTask (zone.js:401:60)
    at Zone.runTask (zone.js:171:47)
    at drainMicroTaskQueue (zone.js:581:35)
```

When quitting the app or closing the `OS default` tab, I get this error log. Hope this helps.

**Comment by hunter-xue** [comment]
exit all tabs, and then tabby can be quit correctly.

**Comment by MuhtasimTanmoy** [comment]
Well, I already have the `OS default` tab open, and neither can close that nor the app.

**Comment by Eugeny** [comment]
Fixed in v1.0.204

**Comment by csrissettled** [comment]
one workaround, kill exploerer.exe and re-run it , for windows version
